Failing Withings Test

<!-- READ THIS FIRST:
  - If you need additional help with this template, please refer to https://www.home-assistant.io/help/reporting_issues/
  - Make sure you are running the latest version of Home Assistant before reporting an issue: https://github.com/home-assistant/home-assistant/releases
  - Do not report issues for integrations if you are using custom components or integrations.
  - Provide as many details as possible. Paste logs, configuration samples and code into the backticks.
  DO NOT DELETE ANY TEXT from this template! Otherwise, your issue may be closed without comment.
-->
## The problem
<!-- 
  Describe the issue you are experiencing here to communicate to the
  maintainers. Tell us what you were trying to do and what happened instead.
-->
when I try to push a PR, it fails on a tests in the withings component, which I don't use.
I tested it as well on a vanilla copy of dev with no changes and it still fails
This is probably blocking any PR from completing the checks successfully

## Environment
<!--
  Provide details about the versions you are using, which helps us to reproduce
  and find the issue quicker. Version information is found in the
  Home Assistant frontend: Developer tools -> Info.
-->

- Home Assistant release with the issue: latest from dev branch
- Last working Home Assistant release (if known): 
- Operating environment (Hass.io/Docker/Windows/etc.): both in my linux venv and in the CI
- Integration causing this issue: withings (probably - their test fails)
- Link to integration documentation on our website: 

## Problem-relevant `configuration.yaml`
<!--
  An example configuration that caused the problem for you. Fill this out even
  if it seems unimportant to you. Please be sure to remove personal information
  like passwords, private URLs and other credentials.
-->
N/A happens with the normal pytest
```yaml

```

## Traceback/Error logs
<!--
  If you come across any trace or error logs, please provide them.
-->

```txt
(dynalite_fix) ziv@hass2:~/dynalite_fix/home-assistant$ pytest tests/components/withings/*.py
Test session starts (platform: linux, Python 3.7.5, pytest 5.3.5, pytest-sugar 0.9.2)
rootdir: /home/ziv/dynalite_fix/home-assistant, inifile: setup.cfg
plugins: timeout-1.3.3, cov-2.8.1, aiohttp-0.3.0, requests-mock-1.7.0, sugar-0.9.2
collecting ...
 tests/components/withings/test_common.py ✓✓✓✓                                                                                                                                                                                 31% ███▏

――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― test_data_manager_update_sleep_date_range[pyloop] ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

data_manager = <homeassistant.components.withings.common.WithingsDataManager object at 0x7fdede955f50>

    async def test_data_manager_update_sleep_date_range(
        data_manager: WithingsDataManager,
    ) -> None:
        """Test method."""
        patch_time_zone = patch(
            "homeassistant.util.dt.DEFAULT_TIME_ZONE",
            new=dt.get_time_zone("America/Los_Angeles"),
        )

        with patch_time_zone:
            update_start_time = dt.now()
            await data_manager.update_sleep()

            call_args = data_manager.api.sleep_get.call_args_list[0][1]
            startdate = call_args.get("startdate")
            enddate = call_args.get("enddate")

>           assert startdate.tzname() == "PST"
E           AssertionError: assert 'PDT' == 'PST'
E             - PDT
E             + PST

tests/components/withings/test_common.py:129: AssertionError
----------------------------------------------------------------------------------------------------------- Captured stderr setup -----------------------------------------------------------------------------------------------------------
DEBUG:asyncio:Using selector: EpollSelector
------------------------------------------------------------------------------------------------------------ Captured log setup -------------------------------------------------------------------------------------------------------------
DEBUG    asyncio:selector_events.py:53 Using selector: EpollSelector
----------------------------------------------------------------------------------------------------------- Captured stderr call ------------------------------------------------------------------------------------------------------------
DEBUG:homeassistant.components.withings:Running call.
------------------------------------------------------------------------------------------------------------- Captured log call -------------------------------------------------------------------------------------------------------------
DEBUG    homeassistant.components.withings:common.py:218 Running call.

 tests/components/withings/test_common.py ⨯                                                                                                                                                                                    38% ███▉
 tests/components/withings/test_init.py ✓✓✓✓✓✓✓✓                                                                                                                                                                              100% ██████████

Results (2.12s):
      12 passed
       1 failed
         - tests/components/withings/test_common.py:112 test_data_manager_update_sleep_date_range[pyloop]
(dynalite_fix) ziv@hass2:~/dynalite_fix/home-assistant$

```

## Additional information

--- a/tests/components/withings/test_common.py
+++ b/tests/components/withings/test_common.py
@@ -115,7 +115,7 @@ async def test_data_manager_update_sleep_date_range(
     """Test method."""
     patch_time_zone = patch(
         "homeassistant.util.dt.DEFAULT_TIME_ZONE",
-        new=dt.get_time_zone("America/Los_Angeles"),
+        new=dt.get_time_zone("America/Belize"),
     )
 
     with patch_time_zone:
@@ -126,10 +126,10 @@ async def test_data_manager_update_sleep_date_range(
         startdate = call_args.get("startdate")
         enddate = call_args.get("enddate")
 
-        assert startdate.tzname() == "PST"
+        assert startdate.tzname() == "CST"
 
-        assert enddate.tzname() == "PST"
-        assert startdate.tzname() == "PST"
+        assert enddate.tzname() == "CST"
+        assert startdate.tzname() == "CST"
         assert update_start_time < enddate
         assert enddate < update_start_time + timedelta(seconds=1)
         assert enddate > startdate

